app/assets/stylesheets/show.scss

```diff
@@ -109,10 +117,23 @@
           clear: both;
         }
       }
+
       .resource-files {
         background-color: $body-bg;
-        margin-left: 3rem;
-        width: 90%
+        margin-left: .5rem;
+        margin-right: .5rem;
+
+        tr {
+          td {
+            max-width: 0;
```

Comment:
What is this doing?

Reply:
Without this, the filename cell fills to allow the full text. I'm open to suggestion, I'd have to find where I got that example, but it solved that issue for me.

Reply:
Please document this.

Reply:
I'm a little concerned we're depending on an indeterminate part of the CSS spec that might not be implemented the same in every browser both now and in the future.

Reply:
I'm open to suggestion if you have any ideas on a more idiomatic way to handle this.

Reply:
I added a comment to the css. Not sure if that is sufficient, but I'm not sure where/how else it would make sense to document this change.

Reply:
How about `max-width: 30vw;`?

Reply:
That's looks pretty good actually. Since I'm no css expert, can you please let me know how you came to that?

Reply:
30% of the viewport width is `30vw`.  There's a new thing called container queries, which is really what we want, but the support is only experimental and only in a few browsers: https://caniuse.com/css-container-queries